plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -586,6 +602,13 @@ public static function with_core_profiler_fields( array $plugins ) {
 				'learn_more_link'  => 'https://woocommerce.com/products/meta-ads-and-pixel?utm_source=storeprofiler&utm_medium=product&utm_campaign=freefeatures',
 				'install_priority' => 2,
 			),
+			'woo-update-manager'            => array(
+				'label'            => __( 'WooCommerce.com Update Manager', 'woocommerce' ),
+				'image_url'        => self::get_woo_logo(),
+				'description'      => __( 'Easily manage updates for WooCommerce.com plugins, ensuring security and access to the latest features.', 'woocommerce' ),
+				'learn_more_link'  => 'https://woocommerce.com/posts/introducing-woo-com-update-manager/',
```

Comment:
I wonder if it isn't more future-proof to link to https://woocommerce.com/document/managing-woocommerce-com-subscriptions/#woo-update-manager: the doc is kept up to date, where the blog post is already almost a year old.

Reply:
Neat! 4547886552